Solutions/Corvus.Leasing.Azure.Specs/Corvus.Leasing.Azure.Specs.csproj

```diff
@@ -27,11 +28,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="SpecFlow" Version="3.3.30" />
-    <PackageReference Include="SpecFlow.NUnit" Version="3.3.30" />
-    <PackageReference Include="SpecFlow.Tools.MsBuild.Generation" Version="3.3.30" />
     <PackageReference Include="nunit" Version="3.12.0" />
```

Comment:
I don't think we need this - we get an indirect reference to it from the metapackage.